src/main/resources/liquibase/1.0/changelog-1.0.0.yaml

```diff
@@ -0,0 +1,40 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
```

Comment:
[Optional]:
you can use at least 2 styles:

1. 
fileName: changelog-1.0.0
id: create rating table

2. 
fileName: 20243110746-create-rating-table
id: 20243110746